R/fsi_functions.R

```diff
@@ -62,6 +89,45 @@ fsi_add_fsa <- function(fsi, lvar, tbl) {
   fsi
 }
 
+
+
+#' @title fsi_add_cs
+#' 
+#' @description fsi_add_cs adds the consequent to the FSI model
+#' 
+#' @usage 
+#' 
+#' fsi_add_cs(fsi, lvar, lvals, mfs, bounds)
+#' 
+#' @param fsi The FSI model instantiated with the `fsi_create` function.
+#' @param lvar A character value that represents a linguistic variable of the consequent.
+#' @param lvals A character vector that represents linguistic values of the linguistic variable of the consequent.
+#' @param mfs A character vector of corresponding membership functions.
+#' @param bounds A numeric vector that represents the lower and upper bounds of the consequent domain. 
+#' 
+#' @details 
+#' 
+#' 
+#' @returns 
+#' 
+#' An FSI model populated with a consequent.
+#' 
+#' @example 
+#' 
+#' library(FuzzyR)
+#' 
+#' # Create the vector with the linguistic values of the linguistic variable "visiting experience":
+#' lvals_visiting_exp <- c("awful", "average", "great")
+#' 
+#' # Define the membership function for each linguistic value:
+#' awful_mf <- genmf("trimf", c(0, 0, 20))
+#' average_mf <- genmf("trimf", c(10, 30, 60))
+#' great_mf <- genmf("trapmf", c(40, 80, 100, 100))
+#' 
+#' # Add the consequent to the FSI model:
+#' fsi <- fsi_add_cs(fsi, "visiting experience", lvals_visiting_exp,
```

Comment:
please, you should also include the creation of the `fsi` before adding a consequent into it,